docs/README.md

```diff
@@ -21,6 +21,8 @@ installed. Also install the following libraries:
     # Following is needed only for generating API docs
     $ sudo pip install sphinx
     $ sudo Rscript -e 'install.packages(c("knitr", "devtools", "roxygen2", "testthat"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo Rscript -e 'install.packages(c("rmarkdown"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo pip install pandoc pandoc-citeproc
```

Comment:
(And I say this without knowing much about it --) Isn't this a Python package? would it be preferable to manage via pip if so, since that's cross-platform I think? does the Ubuntu package just install the same thing?